pkgs/by-name/as/asfa/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  stdenv,
+  darwin,
+  gawk,
+  help2man,
+  openssl,
+  pkg-config,
+}:
+let
+  version = "0.10.0";
+in
+rustPlatform.buildRustPackage {
+  pname = "asfa";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "obreitwi";
+    repo = "asfa";
+    rev = "v${version}";
+    hash = "sha256-MnhnwtZmPFhOuiqNiaxJnPu88JOdlpvyVy0YGphblBc=";
+  };
+
+  cargoHash = "sha256-/bRBP/NzcNOXl/nANeOYouUAo3NNbtbV9fxIJrNajYQ=";
+
+  buildInputs = [
+    openssl
+  ] ++ lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.SystemConfiguration ];
+
+  nativeBuildInputs = [
+    gawk
```

Comment:
```suggestion
```

According to [this](https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/stdenv.chapter.md#tools-provided-by-stdenv-sec-tools-of-stdenv) `awk` is already a part of the default `stdenv`.

(Don't forget to remove it from the top attrset too)

Reply:
Thanks, adapted (& bookmarked).